ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -152,7 +153,9 @@ public void configure(final KsqlConfig config) {
     final Boolean collectMetrics = ksqlConfig.getBoolean(
         KsqlConfig.KSQL_QUERY_PULL_METRICS_ENABLED);
     this.pullQueryExecutorMetrics = collectMetrics
-        ? Optional.of(new PullQueryExecutorMetrics())
+        ? Optional.of(new PullQueryExecutorMetrics(
+            ksqlEngine.getServiceId(),
+            ServiceInfo.create(ksqlConfig, "").customMetricsTags()))
```

Comment:
Since only the custom metrics tags are needed, here we can replace this with simply
```
ksqlConfig.getStringAsMap(KsqlConfig.KSQL_CUSTOM_METRICS_TAGS)
```
rather than worrying about creating ServiceInfo in order to get the metrics tags.